packages/jobs/lib/execution/sync.ts

```diff
@@ -200,6 +201,12 @@ export async function handleSyncSuccess({ nangoProps }: { nangoProps: NangoProps
             environment_id: nangoProps.environmentId,
             provider_config_key: nangoProps.providerConfigKey
         };
+
+        const syncPayload = {
+            scriptVersion: nangoProps.syncConfig.version || 'unknown',
```

Comment:
arguably the script version should probably be displayed in the UI. Maybe something that we can add later

Reply:
yep indeed, it wasn't a thing when I created the schema but we could add `syncVersion`